operator_ui/src/components/Dashboards/Activity.tsx

```diff
@@ -69,20 +79,26 @@ const NoRecentActivity = ({ classes }) => (
   </CardContent>
 )
 
-const RecentActivity = ({ classes, runs }) => {
+interface IProps extends WithStyles<typeof styles> {
+  pageSize: number
+  runs?: any[]
```

Comment:
Can we use the new types here? `(IJobRuns)`

Reply:
Good call